be/src/olap/storage_engine.cpp

```diff
@@ -116,6 +116,7 @@ StorageEngine::StorageEngine(const EngineOptions& options)
         _tablet_manager(new TabletManager()),
         _txn_manager(new TxnManager()),
         _rowset_id_generator(new UniqueRowsetIdGenerator(options.backend_uid)),
+        _memtable_flush_executor(NULL),
```

Comment:
```suggestion
        _memtable_flush_executor(nullptr),
```
prefer nullptr in c++11.